src/Basis/Constructors.jl

```diff
@@ -124,9 +124,9 @@ Transformed quadrature by applying a smooth mapping = (g, gprime) from the origi
 # Example:
 ```jldoctest
 # generate transformed quadrature points, weights with choice of conformal map
-points = LinRange(-1,1,5)
-mapping = haletrefethenstriptransformation(1.4);
-mpoints, mweights = LFAToolkit.transformquadrature(points, true, mapping);
+points, weights = LFAToolkit.gaussquadrature(5)
+mapping = sausagetransformation(9)
+mpoints, mweights = transformquadrature(points, weights, mapping)
```

Comment:
The domain error was because `weights` was incorrectly documented (and used here) as boolean, when in fact it needs to be an array of weights.

I switched to sausage here because it more accurately integrates the constant on coarse grids (few quadrature points); the others need many points to make that true.